packages/tradershub/src/features/cfd/screens/Jurisdiction/jurisdiction-contents/jurisdiction-svg-contents.ts

```diff
@@ -0,0 +1,130 @@
+import { TJurisdictionCardItems } from './props.types';
+
+export const getJurisdictionSvgContents = (): TJurisdictionCardItems => ({
+    contents: {
+        all: [
+            {
+                description: 'Synthetics, Forex, Stocks, Stock Indices, Cryptocurrencies, and ETFs',
```

Comment:
we had content update cards before and the texts here are deprecated, can refer to PR below:

- [CU](https://app.clickup.com/t/20696747/CFDS-1159)
- [Figma](https://www.figma.com/file/7nuUT7T7r29dslMvzTzG2m/Release---V1.0.0---CFDs-compare-accounts-table?type=design&node-id=433-48098&mode=design&t=0FUHGMBvKmpqzYmT-0)
- [PR](https://github.com/binary-com/deriv-app/pull/10345)

Reply:
Let me check that thanks